downloads/platform.md

```diff
@@ -170,6 +170,13 @@ Recent Debian/ubuntu distributions include their own build of Julia, which can b
 sudo apt install julia
 ```
 
+## Arch Linux
+Arch Linux has its [own build of Julia](https://www.archlinux.org/packages/community/x86_64/julia/) in its community repository. To install it run:
+
+```
+sudo pacman -S julia
```

Comment:
Please, no: https://github.com/JuliaLang/www.julialang.org/pull/702#discussion_r402695166

Reply:
Odd, I've never had package-related issues on Arch. I've had plenty on NixOS, but never Arch. If I change it to installing `aur/julia-bin` will that be satisfactory?

Reply:
Does `aur/julia-bin` use the official pre-compiled binaries? If so, yes that would be acceptable to add.

Reply:
Sure does and I've now made the change.